core/src/api/services/plugins.service.js

```diff
@@ -82,5 +91,13 @@ module.exports = {
                     })
                 });
         });
+    },
+    delete : (name)=> {
```

Comment:
why do you have two method for `delete`? if you want to use the `deletePlugin` method you can simply assign it to the delete. `delete : deletePlugin`